frontend/src/components/common/ToastMessage.vue

```diff
@@ -27,42 +22,18 @@ const onError = (error: any, info: string) => {
     if (axios.isAxiosError(error)) {
         const err = error;
         const axiosResponse = err.response;
-        const status = axiosResponse?.status;
-
-        const e401_authenticationErrorMsg = {
-            title: 'Error',
-            text: 'You are not logged in. Please log in.',
-        };
-
-        const e403_authorizationErrorMsg = {
-            title: 'Error',
-            text: 'You do not have the necessary authorization for the requested action.',
-        };
 
-        if (!status) {
-            showToastTopRight(
-                'error',
-                genericErrorMsg.title,
-                genericErrorMsg.text
-            );
-        } else if (status == 401) {
-            showToastTopRight(
-                'error',
-                e401_authenticationErrorMsg.title,
-                e401_authenticationErrorMsg.text
-            );
-        } else if (status == 403) {
-            showToastTopRight(
-                'error',
-                e403_authorizationErrorMsg.title,
-                e403_authorizationErrorMsg.text
-            );
-        } else if (status == 409) {
-            useNotificationMessage.isNotificationVisible = false;
-            useErrorDialog.dialogTitle = axiosResponse.statusText;
-            useErrorDialog.dialogMsg = axiosResponse.data.detail;
-            useErrorDialog.isErrorVisible = true;
-        }
+        const axiosResponseHasDetail = axiosResponse?.data.detail
+            ? axiosResponse.data.detail
+            : genericErrorMsg.text;
+
+        showToastTopRight(
+            'error',
+            axiosResponse ? axiosResponse.statusText : genericErrorMsg.title,
```

Comment:
let's remove the  axiosResponse.statusText, cause status text is not coming back from the server, but from the axios library. We can just use the generic error title here.

Reply:
Ok, using the generic error title.